content/metadata.yaml

```diff
@@ -399,12 +399,12 @@ author_info:
 #   twitter: null
 #   affiliations:
 #   - Distributed Bio
-# - name: Qiwen Hu
-#   orcid: null
-#   github: null
-#   twitter: null
-#   affiliations:
-#   - University of Pennsylvania
+- name: Qiwen Hu
+   orcid: null
```

Comment:
Please try 2 leading spaces instead of 3 in the `github`, `twitter`, and `affiliations` lines.  You can leave the `orcid` line commented or delete it, I'm not certain what will happen with `null`.

Will you also be editing `02.author-sections.md` with your introduction?

Reply:
Thanks a lot !  Yes, I already edited the 02.author-sections.md.